docs/src/gui/qtvcp.adoc

```diff
@@ -145,111 +195,181 @@ Such libraries include:
 
 === Themes
 
-Themes are a way to modify the look and feel of the widgets on the screen.
-For instance the color or size of buttons and sliders can be changed using
-themes.
-The Windows theme is default for screens. System theme is default for panels.
-to see available themes load qtvcp with -d -t SHOWTHEMES .
+Themes are a way to modify the *look and feel* of the widgets on the
+screen.
 
-qtvcp can also be customized with Qt stylesheets using css.
+For instance the _color_ or _size_ of buttons and sliders can be changed
+using themes.
+
+The _Windows theme_ is default for screens. +
+The _System theme_ is default for panels.
+
+To see available themes load with:
```

Comment:
We should not discuss content in this thread, and if it is only because I am long waiting for this to be accepted so the translations can go forward. We just do not have anywhere to discuss documentation without bothering too many with such details.

Reply:
Hopefuly fixed.

@smoe agreed